packages/block-library/src/text-columns/editor.scss

```diff
@@ -3,3 +3,8 @@
 		outline: $border-width solid $gray-300;
 	}
 }
+
+// Hide inserter from previews.
+.block-editor-block-preview__content-iframe .wp-block-columns .block-list-appender {
+	display: none;
+}
```

Comment:
Is there a reason this is being added to the styles of the long-deprecated Text Columns block and not the Columns block?

Reply:
Yes! The reason is I was moving too fast. Ack. I'll address this and Carolina's feedback below.